### PR TITLE
dev-util/clion: version bump to 2017.3.1

### PR DESCRIPTION
**Pull request by strahlc**
* dev-util/clion: version bump to 2017.3.1
* dev-util/clion: remove old version

### CODE REVIEW

File: dev-util/clion/clion-2017.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```
Comment (titanofold) [inline dev-util/clion/clion-2017.3.1.ebuild]: inherit desktop

eutils doesn't have what you need anymore.

Comment (strahlc) [reply dev-util/clion/clion-2017.3.1.ebuild]: sure?
 
why does it work?

``` 
$ equery f clion
 * Searching for clion ...
 * Contents of dev-util/clion-2017.3.1:
...
/usr/bin/clion
/usr/share/applications/clion-clion.desktop
/usr/share/pixmaps/clion.svg
...
```

Comment (titanofold) [reply dev-util/clion/clion-2017.3.1.ebuild]: eutils inherits desktop for existing EAPIs (0 - 6).

When this ebuild is moved to EAPI7, it'll stop working.

Comment (strahlc) [reply dev-util/clion/clion-2017.3.1.ebuild]: alright, added `inherit desktop`.
`inherit eutils` is still required for `make_wrapper`

**Comment by candrews** [comment]
1a599a60f810f513d3e6e78846eb88c983969ee7
c84fe6a17fa1e4343ed5681c95f01931868ebdb1

**Pull request closed without merging**